### PR TITLE
Added system_custom example in testJobTemplate

### LINKED ISSUE #2390: Add documentation to run system_custom
We need to add a documentation somewhere to run `system_custom`. 

The basic content of this documentation is available here: https://github.com/AdoptOpenJDK/openjdk-tests/pull/2234

### PR DESCRIPTION
**Pull request by patkarns**
Fixes #2390

### CODE REVIEW

**Comment by patkarns** [comment]
I tried testing my new changes in WIP_Test_Job_Auto_Gen but it says ERROR: script not yet approved for use
https://ci.adoptopenjdk.net/view/work-in-progress/job/WIP_Test_Job_Auto_Gen/37/console

**Comment by smlambert** [comment]
Yes, script approval is turned on, so it prompts for approval, you should be good now:
https://ci.adoptopenjdk.net/view/work-in-progress/job/WIP_Test_Job_Auto_Gen/39/console

**Comment by patkarns** [comment]
image

I just noticed that my line breaks aren't showing up even though I'm using ''' Line 1 <br> Line 2 <br> Line 3'''

I'll try some more!

File: buildenv/jenkins/testJobTemplate [hunk 1]
```diff
@@ -302,8 +302,14 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							}
 							stringParam('BUILD_LIST', "${ACTUAL_BUILD_LIST}", "Specific test directory to compile, set blank for all projects to be compiled")
 							stringParam('TARGET', "${ACTUAL_TARGET}", "Test TARGET to execute")
-							stringParam('CUSTOM_TARGET', "", "Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, etc., CUSTOM_TARGET=path to the test class to execute")
-							
+							stringParam('CUSTOM_TARGET', "", '''
+								Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, system_custom, etc.,<br>
+								Example 1:<br>
+								TARGET=system_custom<br>
+								CUSTOM_TARGET=test/jdk/java/math/BigInteger/LargeValueExceptions.javaCUSTOM_TARGET=path to the test class to execute<br>
```
Comment (smlambert) [inline buildenv/jenkins/testJobTemplate:309]: Can move the end of 309 (CUSTOM_TARGET=path to the test class to execute) up to the end of line 306.  The CUSTOM_TARGET examples are mixed up (the jdk_custom one is listed under example 1, and the system_custom one is listed in example 2).  Need to swap line 309 and line 312.

**Comment by smlambert** [comment]
Latest run generates this:
Screen Shot 2021-04-26 at 9 32 36 PM

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @patkarns !

**Pull request merged**